src/network/os1/client/prot/game/GameClientRepository.ts

```diff
@@ -30,5 +63,31 @@ export default class GameClientRepository extends ClientRepository {
         this.bind(new ClientCheatDecoder(), new ClientCheatHandler())
         this.bind(new IfButtonDecoder(), new IfButtonHandler())
         this.bind(new ClanDecoder(), new ClanHandler())
+        this.bind(new OpPlayer1Decoder(), new OpPlayerHandler())
+        this.bind(new OpPlayer2Decoder(), new OpPlayerHandler())
+        this.bind(new OpPlayer3Decoder(), new OpPlayerHandler())
+        this.bind(new OpPlayer4Decoder(), new OpPlayerHandler())
+        this.bind(new OpPlayer5Decoder(), new OpPlayerHandler())
+        this.bind(new OpPlayer6Decoder(), new OpPlayerHandler())
+        this.bind(new OpPlayer7Decoder(), new OpPlayerHandler())
+        this.bind(new OpPlayer8Decoder(), new OpPlayerHandler())
+        this.bind(new OpNpc1Decoder(), new OpNpcHandler())
+        this.bind(new OpNpc2Decoder(), new OpNpcHandler())
+        this.bind(new OpNpc3Decoder(), new OpNpcHandler())
+        this.bind(new OpNpc4Decoder(), new OpNpcHandler())
+        this.bind(new OpNpc5Decoder(), new OpNpcHandler())
+        this.bind(new OpLocDecoder(), new OpLocHandler())
+        this.bind(new OpLoc2Decoder(), new OpLocHandler())
+        this.bind(new OpLoc3Decoder(), new OpLocHandler())
+        this.bind(new OpLoc4Decoder(), new OpLocHandler())
+        this.bind(new OpLoc5Decoder(), new OpLocHandler())
+        this.bind(new OpObj1Decoder(), new OpObjHandler())
+        this.bind(new OpObj2Decoder(), new OpObjHandler())
+        this.bind(new OpObj3Decoder(), new OpObjHandler())
+        this.bind(new OpObj4Decoder(), new OpObjHandler())
+        this.bind(new OpObj5Decoder(), new OpObjHandler())
+        this.bind(new LocExamineDecoder(), new LocExamineHandler())
```

Comment:
The examine packets are the 6th option with a special engine-case, really called OPLOC6 OPNPC6 OPOBJ6